roles/entitlement_test_wait_deployment/tasks/main.yml

```diff
@@ -15,16 +15,23 @@
       set -o errexit;
       set -o pipefail;
 
+      oc create namespace "{{ entitlement_tester_namespace }}"
+      oc label -n "{{ entitlement_tester_namespace }}" \
+        security.openshift.io/scc.podSecurityLabelSync=false \
+        pod-security.kubernetes.io/enforce=privileged \
+        pod-security.kubernetes.io/audit=privileged \
+        pod-security.kubernetes.io/warn=privileged
+
       oc apply -f "{{ entitlement_tester_entrypoint }}"
       oc delete -f "{{ entitlement_tester_pod }}" --ignore-not-found=true;
       oc create -f "{{ entitlement_tester_pod }}";
 
       i=0;
-      CMD="oc get pod/entitlement-tester
+      CMD=\"oc get pod/entitlement-tester
                 -o custom-columns=:.status.phase
                 --no-headers
-                -n default
-                ";
+                -n "{{ entitlement_tester_namespace }}"
+                \";
```

Comment:
I think you should better escape `-n \"{{ entitlement_tester_namespace }}\"`, it would make the `CMD=` definition easier to read [if that works as expected]